test/TimeTracker.Library.Test/Services/Orchestration/DeleteTests.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using TimeTracker.Data;
+using TimeTracker.Library.Models;
+using TimeTracker.Library.Services;
+using TimeTracker.Library.Services.Orchestration;
+using Xunit;
+
+namespace TimeTracker.Library.Test.Services.Orchestration
+{
+    public class DeleteTests
+    {
+        private readonly TimeTrackerDbContext database;
+        private readonly SlackMessageOrchestrator orchestrator;
+
+        public DeleteTests()
+        {
+            database = new InMemoryDatabaseWithProjectsAndUsers().Database;
+            orchestrator = new SlackMessageOrchestrator(database);
+        }
+
+        [Fact]
+        public async Task HandleCommand_deleteHours_returnsDeletedMessage_andDeletesHours()
+        {
+            var user = database.Users.First();
+            var timeEntryService = new TimeEntryService(user.UserId, database);
+            await timeEntryService.CreateBillableTimeEntry(DateTime.UtcNow.Date, 7, 1, 1);
```

Comment:
what if person has more than one entry? we should make sure delete removes all entries for day...

Reply:
We just refactored this test to be in a separate class. We can add this functionality for sure, but that's not really part of the bugfix.

Reply:
that's cool, just wanted to be sure to address something when i noticed it